okcurl/src/jvmTest/kotlin/okhttp3/curl/MainTest.kt

```diff
@@ -18,26 +18,32 @@ package okhttp3.curl
 import java.io.IOException
 import okhttp3.RequestBody
 import okio.Buffer
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.Test
-import picocli.CommandLine
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNull
+import assertk.assertions.startsWith
+//import kotlin.test.Test
+//import kotlin.test.fail
 
 class MainTest {
-  @Test fun simple() {
+//  @Test
```

Comment:
Can't work out why the build is failing on this package of all things...

Reply:
Adding `kotlin("test")` to `commonTest` works for me (see https://github.com/yschimke/okhttp/pull/18). Not really sure how it works. I guess it's a special maven artifact that pulls the various implementations for different platforms